documentation/api/query/curl.markdown

```diff
@@ -82,6 +82,18 @@ If you do this with an endpoint that accepts `GET` requests, **you must also use
     curl -G http://localhost:8080/pdb/query/v4/nodes \
       --data-urlencode 'query=["=", ["node", "active"], true]'
 
+## Pretty querying of PuppetDB
+
+PuppetDB returns unprettified JSON by default. PuppetDB provides the option of
```

Comment:
Does this need to be accompanied with a "deprecation" of pretty by default on the 3.x side?